packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -77,4 +78,26 @@ export const library = [
       ),
     ],
   }),
+  maker.make({
+    name: "joint",
+    output: "Plot",
+    examples: [
+      `Plot.joint({ xDist: 2 to 5, yDist: SampleSet.fromDist(-3 to 3) })`,
```

Comment:
Ideally the syntax could later work for Lists too. That said, starting with longer names should be okay now, can adjust later.